rslib/src/preferences.rs

```diff
@@ -149,3 +155,23 @@ impl Collection {
         Ok(())
     }
 }
+
```

Comment:
A bit boilerplaty. Should I ditch the native struct in favour of the auto-generated pb one?

Reply:
We could probably just reuse the pb here; it's a simple struct and a native copy of it doesn't buy us anything.